ios/Codeforces Watcher/Application/features/users/UserViewController.swift

```diff
@@ -48,16 +48,35 @@ class UserViewController: UIViewControllerWithCross {
         navigationItem.rightBarButtonItem = UIBarButtonItem(
             image: UIImage(named: "removeIcon"), 
             style: .plain, 
-            target: self, 
-            action: #selector(removeTapped)
+            target: self,
+            action: #selector(showAlertToDeleteUser)
         )
         
         buildViewTree()
         setConstraints()
         bind()
     }
     
-    @objc func removeTapped() {
+    @objc func showAlertToDeleteUser() {
+        let alertController = UIAlertController(
+            title: "Delete user",
```

Comment:
For strings to be localized (translated to other languages), they need to be moved to `Base.lproj/Localizable.strings`. Than you can use `"string".localized` and `"string".localizedFormat` on them to apply that translations.

Reply:
Do I need to add translations to other languages too?

Reply:
Well, right now - not 😉 We will add it before releasing the app.